examples/common/timestamp/timestamp-utc.yaml

```diff
@@ -0,0 +1,13 @@
+---
+id: timestamp-utc
+type: Feature
+geometry:
+  type: Point
+  coordinates: [0, 0]
+properties:
+  categories:
+    main: someCategory
+  theme: places
+  type: place
```

Comment:
For variety, maybe make this a different theme and feature type? This could be transportation/connector.

Reply:
I would prefer to keep these equal to keep the focus on the timestamp values. This is not a strong opinion and I have no issue in changing feature type. But just to clarify, what is the value of using different feature type?

Reply:
The idea is it increases your coverage a bit because it tests that the rules work across feature types. The way the JSON Schema is structured is a bit janky (and can change), so it's easy enough to break something on one feature type but not another one.

You could argue that the difference in coverage is minute, and you might be right. ... But it makes me feel better :)

Reply:
Done